cylc/rose/utilities.py

```diff
@@ -132,16 +132,19 @@ def get_rose_vars_from_config_node(config, config_node, environ):
         config['env'] = {
             item[0][1]: item[1].value for item in
             config_node.value['env'].walk()
+            if not item[1].state
```

Comment:
The states are stored as constants in Rose, would be better to filter for `STATE_NORMAL`:

```suggestion
            if item[1].state == ConfigNode.STATE_NORMAL
```